pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -74,6 +74,7 @@ func NewCmdRollingUpdate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().String("timeout", timeout, `Max time to wait for a controller to update before giving up. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to use to create the new controller.")
 	cmd.Flags().String("image", "", "Image to upgrade the controller to.  Can not be used with --filename/-f")
+	cmd.Flags().String("container-name", "", "Container name which will have its image upgraded. Only relevant when --image is specified, ignored otherwise. Required when using --image on a multi-container pod")
```

Comment:
Please conform to the convention of other commands, such as log and exec: `--container`, `-c`
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/log.go#L85
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/exec.go#L68